onnxruntime/server/executor.cc

```diff
@@ -32,19 +31,22 @@ protobufutil::Status Executor::SetMLValue(const onnx::TensorProto& input_tensor,
   auto logger = env_->GetLogger(request_id_);
 
   size_t cpu_tensor_length = 0;
-  auto status = onnxruntime::server::GetSizeInBytesFromTensorProto<0>(input_tensor, &cpu_tensor_length);
-  if (!status.IsOK()) {
-    logger->error("GetSizeInBytesFromTensorProto() failed. Error Message: {}", status.ToString());
-    return GenerateProtobufStatus(status, "GetSizeInBytesFromTensorProto() failed: " + status.ToString());
+  try {
```

Comment:
Why don't we use the return value here? try-catch will hurt the performance.

Reply:
GetSizeInBytesFromTensorProto doesn't seem to return anything.

Reply:
The c++ api uses exceptions, so I decided to use exceptions for the converter as well. IIRC c++ exceptions are generally zero cost in the happy path so this will only hurt performance when it fails.